router/internal/recoveryhandler/recovery.go

```diff
@@ -62,41 +33,6 @@ func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	defer func() {
 		if err := recover(); err != nil {
 			w.WriteHeader(http.StatusInternalServerError)
-
```

Comment:
This has all basically been moved into the logger defer?

Reply:
Yes, recovery only recovers and sets the status code.